Web/Admin.UI/src/app/components/monitor/link-health-check/link-health-check.component.html

```
@@ -1,96 +1,89 @@
 <div class="content" *ngIf="!initializeSummaries;else initial_summary_loading">
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)